components/brave_wallet/brave_wallet_service.cc

```diff
@@ -166,8 +168,15 @@ void BraveWalletService::SaveToPrefs(PrefService* prefs,
 }
 
 void BraveWalletService::ResetCryptoWallets() {
+  Profile* profile = Profile::FromBrowserContext(context_);
```

Comment:
I saw a few places also checking whether or not this is a Tor profile  -- do we care about that in this context? (I guess the question is should we enable Wallets reset in a Tor profile). Here's one example:

https://github.com/brave/brave-core/blob/bfca25a6975f7ef0824d38f48fb81b997db6da26/browser/extensions/api/brave_wallet_api.cc#L96-L100

I don't have a strong opinion one way or another, just curious!

Reply:
Good catch, I'll add a guard for the reset method in `brave_wallet_api.cc`